### PR TITLE
[orc8r] Adding api to provide total number of subscribers

### LINKED ISSUE #5892: [Orc8r] Additional api endpoints for supporting subscriber paging
### Your Environment

- **Version:**
- **Affected Component:** Orchestrator, Access Gateway, or NMS

### Describe the Issue

Need for 3 additional API endpoints to support subscriber paging
- subscriber state paging endpoint
- total subscriber count endpoint
- total subscriber state count endpoint

### PR DESCRIPTION
**Pull request by karthiksubraveti**
## Summary
Added total count attribute to existing subscriber v2 api

## Test Plan
Changed existing test to check for total count. 

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go
Comment (andreilee) [inline lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go]:
```suggestion
	totalCount := swag.int64(count)
```

File: orc8r/cloud/go/services/configurator/client_api.go
Comment (hcgatewood) [inline orc8r/cloud/go/services/configurator/client_api.go]: "CountEntitiesOfType"

File: lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go
Comment (hcgatewood) [inline lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go]: Let's not make a configurator call here, since we may be making an early return if ParamMSISDN or ParamIP are set. Move the configurator call below those

File: lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml
Comment (hcgatewood) [inline lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml:376]: I'm fine with the way we're doing this count (making it a separate call to configurator) for the sake of simplicity of implementation. That said, if we do keep it as a separate configurator call, this field needs to be `estimated_total_count`, to indicate that it's not a reliable count of exactly how many subscribers there are

File: orc8r/cloud/go/services/configurator/storage/sql_entity_load_helpers.go
Comment (hcgatewood) [inline orc8r/cloud/go/services/configurator/storage/sql_entity_load_helpers.go]: "countEntities"

File: orc8r/cloud/go/services/configurator/storage/sql_entity_load_helpers.go
Comment (hcgatewood) [inline orc8r/cloud/go/services/configurator/storage/sql_entity_load_helpers.go]: This is duplicating a lot of logic. It's basically the same as the normal builder, but with "select count" instead of "select rows". I think there's a way to work those together

That said, wait for this PR until we get the configurator changes merged. Cause it will change things enough that you'll need to rework just a bit

**Review by hcgatewood — CHANGES_REQUESTED** [review]
Let's wait for https://github.com/magma/magma/pull/5928 to merge this -- so we can reuse the query builder, just with "select count" instead"

File: lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml
Comment (karthiksubraveti) [reply lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml:376]: I don't think we need to be extremely precise in naming here. Calling this estimated_total_count maybe more confusing rather than clarifying the implementation detail involved here. Since it is more an informational field in UI, I would prefer ignoring this detail and just calling this total_count.

File: orc8r/cloud/go/services/configurator/storage/sql_entity_load_helpers.go
Comment (hcgatewood) [inline orc8r/cloud/go/services/configurator/storage/sql_entity_load_helpers.go]: Make this one big switch on loadTyp -- right now it's two separate if statements

File: lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml
Comment (hcgatewood) [inline lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml]: Fine with leaving it as `total_count`, but then let's note the fact that it's an estimation, or best-effort count, in the description

File: lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml
Comment (hcgatewood) [inline lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml:378]: Since total_count is a required property, make it `x-nullable: false`

**Review by hcgatewood — APPROVED** [review]
Lgtm, please handle 3x notes before merging

**Pull request merged**
